_component_design/popovers.md

```diff
@@ -5,90 +5,180 @@ secondary: Editing
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Popover redline specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-specs.svg){:width="80%"}
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Closeable popovers are triggered as a result of...
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Pointer size and placement
+
+
+The pointer is 16 px wide at the base. The apex is 8 px from the side of the rectangle. The pointer should start 12 px from the corner. The side from which the pointer protrudes depends on the location of the trigger on the screen.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+- Always have the popover protrude toward the center of the screen.
+- Do not hide important information with the popover.
+- The popover content should be repositioned so that it doesn't bleed off the screen.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Closeable popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Pointer specifications and pointer placement variations"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-pointer-size-placement.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Closeable popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+## Types of popovers
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Small create or edit popover
+
+A popover may be used for small create/edit workflows that need to remain in-context.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover with a small create workflow"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-small-create-edit-popover.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Closeable popover w/ header"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable-header.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Large hidden content
+
+A popover may be used to reveal hidden content, so the information is still accessible, but doesn’t clutter the screen.
```

Comment:
You can use a popover to reveal hidden content...
remove commas

Reply:
updated.